deploy/chart/templates/deployment.yaml

```diff
@@ -15,29 +15,46 @@ spec:
     metadata:
       labels: {{- include "chart.selectorLabels" . | nindent 8 }}
     spec:
-    {{- with .Values.imagePullSecrets }}
+      {{- with .Values.imagePullSecrets }}
       imagePullSecrets: {{- toYaml . | nindent 8 }}
-    {{- end }}
+      {{- end }}
       serviceAccountName: {{ include "chart.fullname" . }}
-      securityContext: {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- with .Values.podSecurityContext }}
+      securityContext: {{- toYaml . | nindent 8 }}
+      {{- end }}
       containers:
         - name: {{ .Chart.Name }}
           securityContext: {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:v{{ .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            {{- if .Values.clamav.enable }}
+            - --engine-clamav-agent-image
+            - {{ .Values.clamav.agent.image.repository }}:v{{ .Chart.AppVersion }}
+            - --engine-clamav-updater-image
+            {{- with .Values.clamav.updater }}
+            - {{ .image.repository }}:v{{ $.Chart.AppVersion }}
```

Comment:
Apparently the AppVersion is already prefixed with a `v`, we should either use `.Chart.Version` or just drop the prefix here. https://github.com/mittwald/kube-av/blob/383066809555bf5880dc9a3f751d2d8ecf5877ec/deploy/chart/Chart.yaml#L6

Reply:
Good catch! :+1: